services/inference_service.py

```
@@ -0,0 +1,88 @@
+# 모델 파일 경로
+MODEL_PATH = "models/intent_model.pkl"
+
+# 모델 버전
+MODEL_VERSION = "mock-v2"
+
+
+def load_model():
+    # pkl 파일이 존재하면 모델 로드
+    if os.path.exists(MODEL_PATH):
+        with open(MODEL_PATH, "rb") as file:
+            model = pickle.load(file)
```

Ruff (0.15.11)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)